samples/cpp/text_skewness_correction.cpp

```diff
@@ -0,0 +1,61 @@
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+
+#include <iostream>
+#include <string>
+using namespace cv;
+using namespace std;
+
+
+
+int main( int argc, char** argv )
+{
+    CommandLineParser parser(argc, argv, "{@input | imageTextR.png | input image}");
+
+    // Load image from the disk
+    Mat image = imread( samples::findFile( parser.get<String>("@input") ), IMREAD_COLOR);
+    if(image.empty())
+    {
+        cout<<"Cannot load the image " + parser.get<String>("@input") << endl;
+        return -1;
+    }
+
+    Mat gray;
+    cvtColor(image, gray, COLOR_BGR2GRAY );
+    bitwise_not(gray, gray);
```

Comment:
you can remove this line and use
`threshold(gray, thresh, 0, 255, THRESH_BINARY_INV | THRESH_OTSU);` below